jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TravelerMessageBuilder.java

```diff
@@ -1,188 +1,206 @@
-//package us.dot.its.jpo.ode.traveler;
```

Comment:
If  we move TravelerInputData to us.dot.its.jpo.ode.plugin.j2735 this class should be moved to us.dot.its.jpo.ode.plugin.j2735.oss and be called OssTravelerInputData.